stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 8.1.1
+version: 8.1.2
```

Comment:
Following semver versioning, you should update to 8.2.0 as you are introducing a new feature in the chart.